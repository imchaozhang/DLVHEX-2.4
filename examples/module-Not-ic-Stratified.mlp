#module(p1,[]).
q(a).
q(b) :- @p2[q]::r(a).


#module(p2,[q/1]).
q(a).
r(a).
s :- @p2[q]::r(a).

