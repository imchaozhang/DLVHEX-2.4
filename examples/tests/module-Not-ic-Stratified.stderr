1 grep -q "Error: not i stratified program"
